modules/core/src/copy.cpp

```diff
@@ -292,7 +292,7 @@ void Mat::copyTo( OutputArray _dst ) const
         {
             // For some cases (with vector) dst.size != src.size, so force to column-based form
             // It prevents memory corruption in case of column-based src
-            if (_dst.isVector())
+            if (_dst.isVector() || _dst.isArray())
```

Comment:
Probably we should avoid introduction of new "isArray()" method and merge it with `isVector()`.